docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -4,36 +4,330 @@
 AWS
 =====
 
+.. _aws-sso:
+
+Using AWS SSO
+-------------
+
+`AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_ (Successor to AWS Single Sign-On, or SSO) is supported.
+
+.. warning::
+  SSO login *will not work across multiple clouds*. If you use multiple clouds, you should :ref:`set up a dedicated user and credentials <dedicated-aws-user>` so that instances launched on other clouds can use AWS resources.
+
+To use it, ensure that your machine `has AWS CLI V2 installed <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_ (by default, ``pip install skypilot[aws]`` installs V1; V2 cannot be installed via pip).
+
+You can use the following to check version:
+
+.. code-block:: console
+
+    $ aws --version
+    aws-cli/2.25.6 ...
+
+Visit your SSO login portal (e.g. ``https://my-sso-portal.awsapps.com/start``), and click on **Access keys** under the corresponding account. Under "AWS IAM Identity Center credentials (Recommended)", copy these values:
+
+- the ``SSO start URL``
+- the ``SSO Region``
+
+Then, log into your SSO account:
+
+.. code-block:: console
+
+    $ aws configure sso
+
+- **SSO session name**: should be set, but you can choose any name you want.
+- **SSO start URL**: copy from the SSO login portal
+- **SSO region**: copy from the SSO login portal
+- **SSO registration scopes**: leave blank to use the default
+
+Log in and approve the request in your web browser. Then back in the CLI, complete the remaining fields:
+
+- **Default client Region**: optional
+- **CLI default output format**: optional
+- **Profile name**: set to ``default`` if you want to use this profile by default with SkyPilot, otherwise see :ref:`several-aws-profiles`.
+
+If everything is set up correctly, :code:`sky check aws` should succeed!
+
+
+.. _dedicated-aws-user:
+
+Setting up a dedicated SkyPilot IAM user
+----------------------------------------
+
+You can create a dedicated IAM user for SkyPilot, if you can't or don't want to use your normal credentials, or if you want to restrict SkyPilot's permissions.
```

Comment:
We should emphasize here that a dedicated user is not needed if `sky check aws` works, and we can put the reason for creating a dedicated IAM user in a itemized list.